File: src/frogger/screen/Controller.scala
package frogger.screen

import java.net.URL
import java.util.ResourceBundle

import frogger.screen.frame.GameFrame
import javafx.fxml.{FXML, Initializable}
import javafx.scene.layout.AnchorPane

class Controller extends Initializable {

    @FXML
    var mainPane: AnchorPane = _

    override def initialize(location: URL, resources: ResourceBundle): Unit = {
      try mainPane.getChildren.add(new GameFrame())
      catch {
        case e: Exception => e.printStackTrace()

      }
    }

}

File: src/frogger/screen/Controller.java
package frogger.screen;

import com.sun.org.apache.xerces.internal.impl.xs.opti.DefaultXMLDocumentHandler;
import frogger.screen.frame.GameFrame;
import javafx.fxml.FXML;
import javafx.fxml.Initializable;
import javafx.scene.input.KeyEvent;
import javafx.scene.layout.AnchorPane;

import java.io.IOException;
import java.net.URL;
import java.util.ResourceBundle;

public class Controller implements Initializable {

    @FXML
    AnchorPane mainPane;

    @Override
    public void initialize(URL location, ResourceBundle resources) {
        try {
            mainPane.getChildren().add(new GameFrame());
        } catch (Exception e) {
            e.printStackTrace();
        }
    }

}